scene/gui/text_edit.cpp

```diff
@@ -1880,6 +1884,14 @@ void TextEdit::_consume_pair_symbol(CharType ch) {
 		}
 	}
 
+	//	Do not need to duplicate quotes while in comments
```

Comment:
Should be a space after `//`, not a tab. Same in line 1895 from previous PR.

Reply:
Fixed, leason learned :)